models/backbone/mobilenet.py

```diff
@@ -0,0 +1,174 @@
+import logging
+import torch.nn as nn
+
+
+from ..utils import conv1x1_group, conv3x3_group, \
+    norm_layer, kaiming_init, constant_init, load_checkpoint
+
+
+class Conv_dw_pw(nn.Module):
+
+    def __init__(self,
+                 inplanes,
+                 planes,
+                 stride=1,
+                 dilation=1,
+                 use_gn=False,
+                 widthmultiplier=1):
+        super(Conv_dw_pw, self).__init__()
+        self.inplanes = round(widthmultiplier * inplanes)
+        self.planes = round(widthmultiplier * planes)
+        self.dw_conv = conv3x3_group(self.inplanes, self.inplanes,
+                                     stride=stride, dilation=dilation,
+                                     groups=self.inplanes)
+        self.pw_conv = conv1x1_group(self.inplanes, self.planes)
+
+        # we want to load pre-trained models
+        # for keep the layer name the same as pre-trained models
+        norm_layers = [norm_layer(self.inplanes, use_gn),
+                       norm_layer(self.planes, use_gn)]
+        self.norm_names = ['bn1', 'bn2'] if not use_gn else ['gn1', 'gn2']
+        for (name, layer) in zip(self.norm_names, norm_layers):
+            self.add_module(name, layer)
+
+    def forward(self, x):
+        # dw
+        x = self.dw_conv(x)
+        norm = getattr(self, self.norm_names[0])
+        x = norm(x)
+        x = nn.ReLU(x)
+
+        # pw
+        x = self.pw_conv(x)
+        norm = getattr(self, self.norm_names[0])
+        x = norm(x)
+        x = nn.ReLU(x)
+        return x
+
+
+def _make_layers(block,
+                 inplanes,
+                 planes,
+                 stride=1,
+                 dilation=1,
+                 use_gn=False,
+                 widthmultiplier=1):
+    layers = [
+        block(
+            inplanes,
+            planes,
+            stride,
+            dilation,
+            use_gn)
+    ]
+    return nn.Sequential(*layers)
+
+
+class MobileNets(nn.Module):
+    r"""PyTorch implementation of the MobileNets architecture
+    <https://arxiv.org/abs/1704.04861>`_.
+    Model has been designed to work on either ImageNet or CIFAR-10
+    Args:
+        num_classes (int): 1000 for ImageNet, 10 for CIFAR-10
+        large_img (bool): True for ImageNet, False for CIFAR-10
+    """
```

Comment:
the `Args` here should be the description of paramters in `__init__` function, please update it